src/module/staking/screen/StakingScreen.tsx

```diff
@@ -0,0 +1,15 @@
+import { ScrollView } from "@peersyst/react-native-components";
+import BaseMainGradientScreen from "module/main/component/layout/BaseMainGradientScreen/BaseMainGradientScreen";
+import StakingSlider from "module/staking/component/core/StakingSlider/StakingSlider";
+
+const StakingScreen = (): JSX.Element => {
+    return (
+        <BaseMainGradientScreen>
```

Comment:
Aquí amb la `ScrollView` no es vue bé. Millor si fas un BaseComponent tant per la HomeScreen com per la StakingScreen que comparteixen força cosa

Reply:
He fet un nou approach a veure si et mola! 👀